book/website/community-handbook/bookdash.md

```diff
@@ -7,7 +7,8 @@ They also contribute to enhancing the project by improving the ways we work in t
 In the past, we have organised 1-2 days long Book Dash events in person or in a hybrid format, where one of the participants coordinated with their team remotely. 
 However, to ensure that international participants have an equal chance to join and address the challenges of hosting in-person events during the COVID-19 pandemic, we started hosting Book Dashes virtually. 
 These virtual Book Dashes are five days long, designed for flexible participation by members in different time zones. 
-Meaning, rather than committing their entire working day, participants can choose one or multiple 2.5 hours short co-working sessions each day based on their availability.
+Meaning, rather than committing their entire working day, participants can choose one or multiple 2.5 hours short collaborative co-working calls, called contribution sessions each day based on their availability.
```

Comment:
```suggestion
Meaning, rather than committing their entire working day, participants can choose one or multiple 2.5 hours short collaborative co-working calls, called "contribution sessions" each day based on their availability.
```
Just adding this suggestion, if this is what they are officially called :)

Reply:
Good catch Anne.